### ISSUE TITLE
ReactiveKit + Bond under Swift 4

### ISSUE THREAD

**Issue by svachmic** [ReactiveKit/Bond#467]
I'm getting a dead-lock with dependecies among Bond and ReactiveKit:

```
[!] Unable to satisfy the following requirements:

- `ReactiveKit (~> 3.7)` required by `Podfile`
- `ReactiveKit (~> 3.6.2)` required by `Bond (6.3.0)`
```

Is there a workaround for this or not?

**Comment by srdanrasic** [comment]
You could specify 3.6.2 in your Podfile.

However, I've just released Bond 6.4.0 that works with ReactiveKit v3.7.x. I suggest you update to that so you're on latest everything (assuming you are on Xcode 9) :)

**Comment by svachmic** [comment]
Yeah, I'm already on Xcode 9.

6.4.0 works great. Thanks a lot, you rock ;)
